README.md

```diff
@@ -66,13 +70,17 @@ Symbol                              | NET35              | NET40              |
 
 * `FEATURE_APPDOMAIN` - enables support for features that make use of an AppDomain in the host.
 * `FEATURE_ASSEMBLYBUILDER_SAVE` - enabled support for saving the dynamically generated proxy assembly.
+* `FEATURE_COM` - enables support for COM Interop.

```

Comment:
The description isn't quite accurate since all it does is enable the COM unit tests.